maitake/src/scheduler.rs

```diff
@@ -204,7 +204,7 @@ impl Core {
         };
 
         for task in self.run_queue.consume() {
-            let _span = debug_span!("poll", ?task).entered();
+            let _span = debug_span!("poll", ?task, task.tid = %task.id()).entered();
```

Comment:
Would it make sense to have an "as integer" method here? Otherwise we'll need to do `atoi` every time to format the task ID into a string.